### PR TITLE
Add spelling CI step

### PR DESCRIPTION
**Pull request by timgates42**
Initially including existing mistakes to check it fails - then including spelling fixes.

### CODE REVIEW

**Comment by timgates42** [comment]
@garyd203 here is the PR I discussed to try out the spell checker in CI

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,3 +2,9 @@
 If you have any feedback, bugs or suggestions, please [open an
 issue](https://github.com/garyd203/flying-circus/issues/new)
 in GitHub.
+
+# Spelling Checks
+As part of CI the spelling of words in any pull request are checked. Aside from
+genuine spelling mistakes, if this fails due to use of a made up word that you
+really want to include because it explainifies the situation precisely then
```
Comment (garyd203) [inline CONTRIBUTING.md:9]: "explainifies" 👍

**Comment by garyd203** [comment]
Thanks @timgates42, I had to rebase your branch to make it commit, in #239, so I will close this PR

**Comment by timgates42** [comment]
Thanks @garyd203

**Pull request closed without merging**